ironfist/src/cpp/combat/creatures.h

```diff
@@ -110,6 +110,8 @@ enum CREATURE_FLAGS : __int16
 struct tag_monsterInfo
 {
   __int16 cost;
+  //__int16 secondary_cost_id;

```

Comment:
A belated apology for ripping into you above erroneously. It took me a while to realize that you hadn't removed things from the structure, and figured this out on your own. Yay for testing!